static/app/components/events/rrwebReplayer/baseRRWebReplayer.tsx

```diff
@@ -0,0 +1,319 @@
+import {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import RRWebPlayer from 'rrweb-player';
+
+import space from 'sentry/styles/space';
+
+type RRWebEvents = ConstructorParameters<typeof RRWebPlayer>[0]['props']['events'];
+
+interface Props {
+  className?: string;
+  events?: RRWebEvents;
+}
+
+const BaseRRWebReplayerComponent = ({events, className}: Props) => {
+  const [playerEl, setPlayerEl] = useState<HTMLDivElement | null>(null);
+
```

Comment:
Previously `index.tsx` contained the styling and the component in `rrWebReplayer` contained this component but also data fetching. I've shifted things around so that the component in `index` handles data fetching and this is the base/styled replayer component.